tao_compiler/mlir/disc/IR/hlo_disc_ops.cc

```diff
@@ -194,6 +194,12 @@ LogicalResult QuantizedDynamicConvOp::verify() {
   return CommonVerifyForQuantizedComputeIntensiveOp(this);
 }
 
+//===----------------------------------------------------------------------===//
+// SparseReshapeOp
+//===----------------------------------------------------------------------===//
+
+LogicalResult SparseReshapeOp::verify() { return Verify(*this); }
```

Comment:
better to add file-check based ut for newly added ops (like the quantization ops).

```
tao_compiler/mlir/disc/IR/tests/...
```

Reply:
done